### ISSUE TITLE
Is there a way access the BlueBird Promise library?

### ISSUE THREAD

**Issue by edgardojimenez** [thuld/CrmFetchKit#4]
Since you are packaging the BlueBird Promise library within CrmFetchKit, I was wondering if there was a way to get the promise object from the CrmFetchKit.  I am trying to use the Promise.All() capability, and don’t want load another library since the CrmFetchKit has it.  

If not, any plans to expose it, or provide the Promise.All() capability within the CrmFetchKit library, like:
```javascript 
CrmFetchKit.all([CrmFetchKit.Fetch(xml1), CrmFetchKit.Fetch(xml2)]).then(function () {
            console.log("all in");
 });
```

**Comment by thuld** [comment]
good point. I just created an question on stackoverflow. Not sure what the best approach is.

http://stackoverflow.com/questions/32616112/browserify-exposing-an-internal-dependency

**Comment by thuld** [comment]
just released version 3.3.0, the promise object is exposed via the "Promise" property. See documentation https://github.com/thuld/CrmfetchKit#promise-member

**Comment by edgardojimenez** [comment]
Thanks, You have done a great job with this library, and made FetchXml easy to use.  I appreciate you hard work.
